### DISCUSSION TITLE
How to set the value above zero?

### DISCUSSION THREAD

**Discussion by weidongzhou1994** [ourownstory/neural_prophet#1413] [Q&A - get help using NeuralProphet]
As the title, the model gives me some negative value, I want to avoid this.

**Comment by leoniewgnr** [comment]
Hi @weidongzhou1994, unfortunately this feature is not implemented yet, but we were thinking about it too! Do you have any ideas on how to implement that? Or would you like to contribute?

**Comment by mmangione** [comment]
I ran into this problem and simply set any values less than 0 to 0 in my resulting predictions. I suspect I know what the problem is, but I need some confirmation.

Here is a sample of the code I used:
```
# Force cutoff of predictions at 0
test_pred_nums = test_pred._get_numeric_data()
test_pred_nums[test_pred_nums < 0] = 0
```
